src/StatsdClient/StatsdUDP.cs

```diff
@@ -78,42 +84,49 @@ internal static IPAddress GetIpv4Address(string name)
 #else
                 IPAddress[] addressList = Dns.GetHostEntryAsync(name).Result.AddressList;
 #endif
-                //The IPv4 address is usually the last one, but not always
-                for(int positionToTest = addressList.Length - 1; positionToTest >= 0; --positionToTest)
+                // The IPv4 address is usually the last one, but not always
+                for (int positionToTest = addressList.Length - 1; positionToTest >= 0; --positionToTest)
                 {
-                    if(addressList[positionToTest].AddressFamily == AddressFamily.InterNetwork)
+                    if (addressList[positionToTest].AddressFamily == AddressFamily.InterNetwork)
                     {
                         ipAddress = addressList[positionToTest];
                         break;
                     }
                 }
 
-                //If no IPV4 address is found, throw an exception here, rather than letting it get squashed when encountered at sendtime
-                if(ipAddress == null)
+                // If no IPV4 address is found, throw an exception here, rather than letting it get squashed when encountered at sendtime
+                if (ipAddress == null)
+                {
                     throw new SocketException((int)SocketError.AddressFamilyNotSupported);
+                }
             }
+
             return ipAddress;
         }
 
-        public void Send(string command)
+        private static string GetHostNameFromEnvVar()
         {
-            SocketSender.Send(MaxUDPPacketSize, command, 
-                encodedCommand => UDPSocket.SendTo(encodedCommand, encodedCommand.Length, SocketFlags.None, IPEndpoint));
-
+            return Environment.GetEnvironmentVariable(StatsdConfig.DD_AGENT_HOST_ENV_VAR);

```

Comment:
I noticed we renamed this constant: https://github.com/DataDog/dogstatsd-csharp-client/pull/111/files#diff-d0ebbcc771918be301e58f78757158cbL15,  should `DD_AGENT_HOST_ENV_VAR` be renamed as well?

Reply:
I renamed `ENTITY_ID_INTERNAL_TAG_KEY` as this field was private.

`DD_AGENT_HOST_ENV_VAR` is a public field and cannot be renamed without breaking client code. Instead, I marked the field as obsolete in this PR:
https://github.com/DataDog/dogstatsd-csharp-client/pull/113/files#diff-d30af27dd1bd06efdef90392c5163883R48-R55